src/Main.php

```diff
@@ -390,12 +390,21 @@ public static function frontScripts()
                 true
             );
 
+            wp_register_script(
+                'selectric_js',
```

Comment:
lets give this a prefix **admwpp-selectric** insread of **selectric_js** this way if on the theme level it created conflicts we can dequew it.